stories/checkbox.stories.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Story, Demo, Description, Code, comment } from './storybook/blocks';
+import './checkbox.scss';
+
+const title = 'Checkbox';
+
+export default { title };
+
+export const Checkbox = () => (
+  <Story>
+    <Demo>
+      <div className="Manifold-Checkbox">
+        <input role="checkbox" type="checkbox" id="checkbox" />
+        <label for="checkbox">Default checkbox</label>
```

Comment:
Not a request-for-change, but just to reinforce the story below it’d be neat to see multiple checkboxes on the demo page, at least as a subconscious “use this when there are a lotta options” nod